zerver/migrations/0227_inline_url_embed_preview_default_off.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-05-31 02:33
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from django.db.backends.postgresql_psycopg2.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def disable_realm_inline_url_embed_preview(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    Realm = apps.get_model("zerver", "Realm")
+    realms = Realm.objects.filter(inline_url_embed_preview=True)
+    realms.update(inline_url_embed_preview=False)
```

Comment:
So, I think this migration doesn't do exactly what we want, in that an organization on a self-hosted server that had enabled inline URL preview would lose that setting.  I suppose we could have a hack that reads `/etc/zulip/settings.py` directly and checks if it matches `INLINE_URL_EMBED_PREVIEW = True`, and if so, skip this step.  I'm not sure if the invasiveness is worth it though given how few people have likely flipped this, so I think probably we should just go with a release notes remark.

Reply:
@timabbott Yes, this is the same concern I had with flipping of the flags for the different emails too. I thought I mentioned that in the PR for that change, but forgot to mention it here. We should add a release notes entry about this for both the flags that we flipped.

Reply:
For digest emails, I didn't bother because I'm not aware of anyone who'd actually turned it on.

Reply:
Fair enough!